src/activity-functions-function-notation.xml

```diff
@@ -662,83 +578,221 @@
               </pg-code>
             </setup>
             <statement>
-              <p>Does <m>2y=x^2-y</m> make <m>y</m> a function of <m>x</m>?</p>
+              <p>Is <m>y</m> a function of <m>x</m> if, <m>y=x^2</m>?</p>
               <p><var name="$eq1" form="buttons" /></p>
-              <p>Does <m>2y=x-y^2</m> make <m>y</m> a function of <m>x</m>?</p>
+              <p>Is <m>y</m> a function of <m>x</m> if, <m>y^2=x</m>?</p>
               <p><var name="$eq2" form="buttons" /></p>
             </statement>
+            <solution>
+              <p>In each case, ask yourself what happens when you choose a value for x?  Do you get one output or do you get more than one output?</p>
+              <p>Is <m>y</m> a function of <m>x</m> if, <m>y=x^2</m>?</p>
+              <p>Yes <m>y</m> is a function of x because for any value of x you pick, there is only one possible <m>y</m> as a result.</p>
+              <p>Is <m>y</m> a function of <m>x</m> if, <m>y^2=x</m>?</p>
+              <p>No <m>y</m> is not function of x because when you solve for <m>y</m> you get the equation <m>y = \pm \sqrt{x}</m></p>
+            </solution>
           </webwork>
         </exercise>
 
-        <p>Earlier we mentioned the word <term>evaluate</term>. Again, this means to find the value of the output given an input value.</p>
+        <p>When looking at a graph it's easy to tell if the graph is that of a function or not.  Remember the inputs are found along the horizontal axis while the outputs are found along the vertical axis.  The rule is the same: one input, one unique output.</p>
+
+        <exercise>
+          <title>Determine if the graph is a function</title>
+          <webwork>
+            <setup>
+              <var name="$nopass">
+                <set>
+                  <member>\text{This graph is a function}</member>
+                  <member correct="yes">\text{This graph is not a function}</member>
+                </set>
+              </var>
+              <var name="$pass">
+                <set>
+                  <member correct="yes">\text{This graph is a function}</member>
+                  <member>\text{This graph is not a function}</member>
+                </set>
+              </var>
+              <pg-code>
+                $nopass = RadioButtons(["This graph is a function","This graph is not a function"],1);
+                $pass = RadioButtons(["This graph is a function","This graph is not a function"],0);
+              </pg-code>
+            </setup>
+            <statement>
+              <p>Drag the input left or right to see the results on the graph.</p>
+              <p>How many outputs does each input yeild?</p>
```

Comment:
Is this a rhetorical question? Just checking that there is not supposed to be an answer blank here.